specs/protocol/rollup-node-p2p.md

```diff
@@ -266,6 +267,10 @@ Canyon/Delta blocks are broadcast on `/optimism/<chainId>/1/blocks`.
 
 Ecotone blocks are broadcast on `/optimism/<chainId>/2/blocks`.
 
+### `blocksv4`
+
```

Comment:
I do not think we need to add a new p2p topic for holocene anymore, so this should be good, waiting on a response: https://github.com/ethereum-optimism/specs/pull/428#issuecomment-2418099412

Reply:
Thanks for flagging that! Correct, @roberto-bayardo responded to your clarification - https://github.com/ethereum-optimism/specs/pull/428#issuecomment-2418259114

Reply:
Note that at the top of the Gossip Topics section it says "There are three topics" but now there are four (we should rephrase that to not specify a number).